frontend/e2e-test/test/e2e/pages/enduser/enduser-navigation.ts

```diff
@@ -191,6 +192,36 @@ export default class EnduserPage {
       )
       .ok()
   }
+
+  private readonly preferredStartDateDatePicker = Selector(
+    '#daycare-application-form .date-picker'
```

Comment:
Using some more specific data-qa selector would be better, this would break if there ever was another c-datepicker added before the desired one.